tests/main/test_predictions_wrapper.py

```diff
@@ -203,8 +217,30 @@ def test_prediction_wrapper_regression(self, dataset_name, housing, energy, diab
         model_predict = model.predict(X_test)
 
         model_wrapper = PredictionsModelWrapperRegression(
-            X_test, model_predict)
+            X_test, model_predict,
+            should_construct_pandas_query=should_construct_pandas_query)
 
         self.verify_predict_outputs(model, model_wrapper, X_test)
         assert not hasattr(model_wrapper, "predict_proba")
         self.verify_pickle_serialization(model, model_wrapper, X_test)
+
+    @pytest.mark.parametrize('length_test_set', [10, 100, 200, 300, 400, 500, 1000])
```

Comment:
this seems like a lot of parameters - maybe only choose 3?  just worried about test time and number of tests run.

Reply:
It doesn't add much to the overall test time. These add a few seconds only.